ovos_date_parser/dates_kab.py

```
@@ -37,42 +47,330 @@
+    time_str = f"d {hour_words.get(hour_12, str(hour_12))}"
+    minute = dt.minute
+
+    # Section 4.1, 4.5: Handle minutes and fractions
+    if minute == 0:
+        pass  # Exact hour, no suffix needed (e.g., "d ttmanya")
+    elif minute == 15:
+        time_str += " u ṛbeɛ"
+    elif minute == 30:
+        time_str += " u neṣṣ"          # Canonical "neṣṣ"
+    elif minute == 45:
+        time_str += " ɣiṛ ṛbeɛ"
```

Ruff (0.15.21)

[warning] String contains ambiguous `ɣ` (LATIN SMALL LETTER GAMMA). Did you mean `y` (LATIN SMALL LETTER Y)?

(RUF001)